utils/pao-profile.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+
+check_for_perf_profile() {
+  if [[ $(oc get performanceprofile --no-headers | awk '{print $1}') == "benchmark-performance-profile-0" ]]; then
+	log "Performance profile already exists"
+	return true
+  else
+        log "Performance profile not found"
+	return false
+  fi
+}
+
+deploy_perf_profile() {
+  # find suitable nodes
+  if [[ "${baremetalCheck}" == '"BareMetal"' ]]; then
+    log "Trying to find 2 suitable nodes only for testpmd"
+    # iterate over worker nodes bareMetalHandles until we have at least 2 
+    worker_count=0
+    workers=$(oc get nodes | grep -v master | grep -v worker-lb | grep -v custom | grep -v NotReady | grep ^worker | awk '{print $1}')
```

Comment:
nit but could we use a line more like this https://github.com/cloud-bulldozer/e2e-benchmarking/blob/528eed64f791ae9856cde8bee082bcd998905c36/workloads/kube-burner/common.sh#L76 which uses labels instead of grep -v'ing everything?